elements/rh-cta/docs/20-guidelines.md

```diff
@@ -0,0 +1,260 @@
+
+### Call to action vs. button
+
+Even though they look similar, a call to action is a link which takes users from one page to a **different page** whereas a button communicates and triggers actions on the **same page**. Use a button when the desired action is submitting a form, canceling a process, or creating a new object.
+
+## Variants
+
+Each call to action has a specific function and the design of each variant signals that function to users. Therefore, it is important that each variant is implemented consistently so they communicate the correct actions.
+
+{% example palette="light",
+            alt="Image of the Primary, Secondary, Brick, and Default variants with descriptive text labels below",
+            src="../cta-variants.png" %}
+
+| Variant     | Use case                                                                                                     |
+| ------------| -------------------------------------------------------------------------------------------------------------|
+| Primary     | Use only once per page for the primary or most important link                                                |
+| Secondary   | Use several times on the same page for secondary or general links                                            |
+| Brick       | Use several times on the same page to group links together in a grid                                         |
+| Default     | Use several times on the same page for tertiary or less important links                                      |
+| Video       | Triggers a video in a [dialog](/elements/dialog/) (applies to Primary, Secondary, and Default variants only) |
+
+{.full-width .col-13}
+
+### Disabled
+
+To indicate that a link is currently unavailable, calls to action can become disabled.
+
+{% example palette="light",
+            alt="Image of disabled Primary, Secondary, Brick, and Default variants",
+            src="../cta-disabled.png" %}
+
+### Brick icons
+
+When grouping Brick variants, icons can be paired with text labels to communicate various topics, but the icons must be the same color, height, style, etc.
+
+{% example palette="light",
+            alt="Image of Brick variants with an icon on the left of text and on top of text",
+            src="../cta-brick-icons.png" %}
+
+## Writing content
+
+### Call to action text labels
+
+Call to action text labels should be more action-oriented than button text labels. When writing text labels, be specific and clearly communicate the task or destination by doing the following:
+- Make sure the text is unique and easily understood for when screen readers announce text to users
+- Always start the text label with a verb
+- Use simple words and phrases
+- Do not use multiple verbs
+- Try to write short text labels or rearrange words when possible
+- Do not use punctuation
+- Do not use any other icons except for the ones that are included
+
```

Comment:
```suggestion

<br>

```

Reply:
The following image is indented.

Reply:
We should use a class in the example shortcode or the style attribute, but ideally, not make exceptions for things like this

Reply:
I'm still seeing the image indented. Looking at the markup, it appears the image is part of the last list item when it shouldn't be: https://ux.redhat.com/elements/call-to-action/guidelines/#call-to-action-text-labels.

I had this same problem in pagination. I looked at some markdown references, but none of the recommendations worked (like adding two or more spaces at the end of the last list item to end the list). The only solution I found was adding a `<br>` in between...I know #shame-on-me.

Reply:
Shame is not called for; all technology is just someone else' mess. 

https://github.com/RedHat-UX/red-hat-design-system/pull/1085